### DISCUSSION TITLE
Evennia translations work?

### DISCUSSION THREAD

**Discussion by drago73** [evennia/evennia#2877] [Evennia Questions] [answered]
hi, the Evennia translations work fine? I have set my lenguage but not all messages from server is traslate...appear half english and half my lenguage.
                           en                                       it
Command 'u' is not available. Digita "help" per ottenere aiuto.

**Comment by Griatch** [comment, marked as answer]
Most likely your language translation is missing strings. Translations are kept up-to-date by volunteers. If you want to help out, check out [The Translation issue](https://github.com/evennia/evennia/issues/2884).
